test/server-test.js

```diff
@@ -0,0 +1,38 @@
+const assert  = require('chai').assert
+const app     = require('../server')
+const request = require('request')
+
+describe('Server', () => {
```

Comment:
Be careful with arrow functions in `mocha` environments. If you get weird errors involving the request, port, or anything about `this`, consider switching to `function` syntax.